frontend/src/App.tsx

```diff
@@ -71,11 +72,13 @@ const App = (): React.ReactElement => {
                 exact
                 path={Routes.DISPLAY_ENTITY_PAGE}
                 component={DisplayPage}
+                blockedRoles={new Set([UserRole.VOLUNTEER, UserRole.STAFF])}
```

Comment:
I think having the allowed roles was better for readability since it would make it easier to quickly tell that this is an Admin only page. Also, most of the pages in our app that require this feature would be Admin only.

Reply:
ah okay that's a good point! lemme roll back my last commit then haha